docs/developers/config_tin.py

```diff
@@ -0,0 +1,76 @@
+from dataclasses import dataclass
```

Comment:
This feels out of place, did you mean to have it here?  Is is just a reference?

Reply:
Is this not the way? See the first code line shown at https://docs.python.org/3/library/dataclasses.html